tools/slo-generator/tests/unit/test_cli.py

```diff
@@ -15,22 +15,55 @@
 import os
 import unittest
 
-from slo_generator.cli import parse_args
+from mock import patch
+from slo_generator.cli import parse_args, cli
+from test_stubs import CTX, mock_sd
 
 cwd = os.path.dirname(os.path.abspath(__file__))
+root = os.path.dirname(os.path.dirname(cwd))
 
 
 class TestCLI(unittest.TestCase):
 
+    def setUp(self):
+        for k, v in CTX.items():
+            os.environ[k] = v
+
     def test_parse_args(self):
         slo_config_path = f'{cwd}/fixtures/slo_config.json'
         error_budget_policy_path = f'{cwd}/fixtures/error_budget_policy.json'
```

Comment:
This file was deleted?

Reply:
This test just verifies that the parse filepath correspond to the initial one (tests the `parse_args` method). I've updated the path to `/path/to/slo_config.json`.